app/main/controllers/template/rtmedia-functions.php

```diff
@@ -3163,25 +3163,25 @@ function rtm_get_music_cover_art( $media_object ) {
 	$media_tags = new RTMediaTags( $file );
 	$title_info = $media_tags->title;
 	$image_info = $media_tags->image;
-	$image_mime = $image_info['mime'];
-	$mime       = explode( '/', $image_mime );
 	$id         = $media_object->id;
 
-	if ( ! empty( $image_info['data'] ) ) {
+	if ( ! empty( $image_info['data'] ) && ! empty( $image_info['mime'] ) ) {
+		$mime = explode( '/', $image_info['mime'] );
 
-		$thumb_upload_info = wp_upload_bits( $title_info . '.' . $mime[ count( $mime ) - 1 ], null, $image_info['data'] );
-
-		if ( is_array( $thumb_upload_info ) && ! empty( $thumb_upload_info['url'] ) ) {
-			$media_obj->model->update(
-				array(
-					'cover_art' => $thumb_upload_info['url'],
-				),
-				array(
-					'id' => $id,
-				)
-			);
+		if ( ! empty( $mime ) && is_array( $mime ) ) {
+			$thumb_upload_info = wp_upload_bits( $title_info . '.' . $mime[ count( $mime ) - 1 ], null, $image_info['data'] );
+			if ( ! empty( $thumb_upload_info['url'] ) ) {
+				$media_obj->model->update(
+					array(
+						'cover_art' => $thumb_upload_info['url'],
+					),
+					array(
+						'id' => $id,
+					)
+				);
 
-			return $thumb_upload_info['url'];
+				return $thumb_upload_info['url'];
+			}
```

Comment:
Else return false

Reply:
There's a code which is getting executed after this block. So I think returning false in else isn't a good idea.